src/database/Match.js

```diff
@@ -47,6 +47,22 @@ export default class Match extends Record({
     }
   }
 
+  getShortName() {
```

Comment:
You can use `getDisplayName(short=true)` instead.

Reply:
See below.